src/main/java/core/basesyntax/garage/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.garage;
+
+public class Bulldozer extends Machine{
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started its work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer stopped working");
```

Comment:
The message should be more informative and indicate that it is a 'Bulldozer' that stopped working. For example: 'Bulldozer has stopped working.'